src/query-client/routines/member.js

```diff
@@ -0,0 +1,4 @@
+import createRoutine from './utils';
+
+export const signOutRoutine = createRoutine('SIGN_OUT');
+export const editMemberRoutine = createRoutine('EDIT_MEMBER');
```

Comment:
Do you need these?

Reply:
I will need them in the future but for now I will delete them